### DISCUSSION TITLE
Vercel not deploying app from my github repository

### DISCUSSION THREAD

**Discussion by maumaumaumaumaumau** [vercel/vercel#5446] [Help] [answered]
Hi, I've  just edited my code today and it doesn't seem to deploy, tried using commit again and it didn't. There is no log of deployment neither queued. I have vercel with read access to all of my repositories, why is this happening?

**Comment by mcsdevv** [comment, marked as answer]
Hello @maumaumaumaumaumau, there could be many reasons for this, can you share the repository so others can take a look?
